packages/theme-patternfly-org/templates/mdx.js

```diff
@@ -126,10 +126,7 @@ export const MDXTemplate = ({
 
   return (
     <React.Fragment>
-      <PageSection
-        id={isSinglePage ? 'main-content' : 'nav-content'}
-        type={isSinglePage ? 'default' : 'nav'}
```

Comment:
When we put the "HTML/React/Design" tabs in this section I believe it's technically a nav section. It's fine if we don't want to use it, but in that case I'm not sure why the variant exists at all.